tasks/install-r.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: install | r-version (download)
+  get_url:
+    url: "https://cdn.rstudio.com/r/ubuntu-2004/pkgs/r-{{ item }}_1_amd64.deb" 
```

Comment:
`amd64` can be replaced with `{{ rstudio_connect_machine_map[ansible_machine] }}`. And  `2004` can be probably replace with `{{ ansible_lsb.release replace('.','') }}`.

Reply:
Added.